app/scripts/services/projects.js

```diff
@@ -32,10 +32,13 @@ angular.module('openshiftConsole')
                       return DataService
                               .get('projects', projectName, context, {errorNotification: false})
                               .then(function(project) {
```

Comment:
Here is where returning the rules from `AuthorizationService.getProjectRules()` could be useful:
```javascript
AuthorizationService 
  .getProjectRules(projectName)
  then(function resolve(rules) {
    return [project, context, rules]; // might be useful to just get the rules here
  });
```